Makefile

```diff
@@ -1,6 +1,27 @@
 .PHONY: all
 all: test install ## build APIKit and run tests
 
+PACKAGE_ROOT = github.com/ExperienceOne/apikit
+PACKAGE_VERSION = ${PACKAGE_ROOT}/internal/framework/version
+
+GIT_COMMIT=""
+GIT_BRANCH=""
+GIT_BRANCH_CLEAN=""
```

Comment:
@donutloop `GIT_BRANC_CLEAN=""` isn't used within the Makefile and can be removed

Reply:
I gonna remove this unused variable from this makefile.

Reply:
@seseloos done